.circleci/config.yml

```diff
@@ -2,23 +2,44 @@ version: 2.1
 
 orbs:
   hmpps: ministryofjustice/hmpps@3.6
+  ruby: circleci/ruby@1
 
 parameters:
   alerts-slack-channel:
     type: string
     default: ppud-replacement-devs
 
+jobs:
+  test:
+    docker:
+      - image: cimg/ruby:2.7-node
+    environment:
+      CIRCLE_CI_AUTH_TOKEN: foobar
```

Comment:
🤔  is that right?

Reply:
We need a token, but as we're using stubbed data it can be a fake :)